src/zope/app/container/browser/tests/test_contents_functional.py

```diff
@@ -88,9 +88,7 @@ def test_inplace_add(self):
             form={'type_name': u'BrowserAdd__zope.site.folder.Folder'})
         body = response.unicode_normal_body
         self.assertIn('type="hidden" name="type_name"', body)
-        #self.assertIn('input name="new_value"', body)
         self.assertIn('type="submit" name="container_cancel_button"', body)
-        #self.assertIn('type="submit" name="container_rename_button"', body)
```

Comment:
The old code had these assertions, so I'm not very happy about their removal.  Were removed because they're now failing?  Why are they failing?

Incidentally, the old code was checking that `container_rename_button` was *not* present, note the `.find(...) < 0` bit.